tests/unit/core/server/testServer.js

```diff
@@ -170,7 +170,7 @@ describe(__filename, () => {
       mockUserApi
         .expects('userProfile')
         .once()
-        .returns(Promise.reject(new Error('example of an API error')));
+        .rejects('example of an API error');
```

Comment:
This should be rejecting with an error instance, not a string

Reply:
👍